report/test/renderer/util-test.js

```diff
@@ -37,8 +37,10 @@ describe('util helpers', () => {
     /* eslint-disable max-len */
     assert.equal(get({formFactor: 'mobile', screenEmulation: {disabled: false, mobile: true}}), 'Emulated Moto G4');
     assert.equal(get({formFactor: 'mobile', screenEmulation: {disabled: true, mobile: true}}), 'No emulation');
+    assert.equal(get({formFactor: 'mobile', screenEmulation: {disabled: true, mobile: true}, channel: 'devtools'}), 'Emulated Moto G4');
     assert.equal(get({formFactor: 'desktop', screenEmulation: {disabled: false, mobile: false}}), 'Emulated Desktop');
     assert.equal(get({formFactor: 'desktop', screenEmulation: {disabled: true, mobile: false}}), 'No emulation');
+    assert.equal(get({formFactor: 'mobile', screenEmulation: {disabled: true, mobile: false}, channel: 'devtools'}), 'Emulated Desktop');
```

Comment:
I thought this would defer to `formFactor`. This situation with `screenEmulation: {disabled: true, mobile: true}` would never happen in devtools:

https://github.com/GoogleChrome/lighthouse/blob/3055496570b7f18378718b2ddcaaa62edde563d7/clients/devtools/devtools-entry.js#L43

Reply:
That's being set on a config extending the default. It gets fully resolved by the time LH runs.

Reply:
Sorry I meant `screenEmulation: {disabled: true, mobile: false}` could never happen. If the config extends the default then `mobile` will always be true.

When I look at a desktop report from DT, `mobile` is still set to `true`.

`devtools-entry.js` does change the `formFactor` property.

Reply:
ah, good catch